app/test/package/backend_test.dart

```diff
@@ -572,11 +572,11 @@ void main() {
           registerAuthenticatedUser(hansUser);
 
           expect(await fn('with'),
-              'Package name must not be a reserved word in Dart.');
+              'PackageRejected(400): Package name must not be a reserved word in Dart.');
```

Comment:
IMO; it would also be fine to refactor this such that `fn` returns the `code` from the response... that way we assert on something less fragile.

Reply:
Good point, I'll take a look in a follow-up PR.